src/mds/DamageTable.h

```diff
@@ -33,71 +31,70 @@ typedef enum
 
 } damage_entry_type_t;
 
+class CDir;
+
```

Comment:
I don't see a need for a forward declaration here. CDir is defined in another file. As you are touching it, replace it with a `#include` ?

Reply:
I have replaced it.